integration/tests/cook/test_cli.py

```diff
@@ -1,18 +1,19 @@
 import json
 import logging
 import os
+import pytest
 import subprocess
 import time
 import unittest
 import uuid
-from urllib.parse import urlparse
 
-from nose.plugins.attrib import attr
+from urllib.parse import urlparse
 
 from tests.cook import cli, util
 
 
-@attr(cli=True)
+@pytest.mark.cli
+@pytest.mark.timeout(600)  # no individual test exceeds 10 minutes
```

Comment:
Is there a way to set this globally instead of on every test class?

Reply:
Nope. 😞

We could probably create our own base class (rather than extending `unittest.TestCase`) and set it once there for all of our classes, but that would be the closest we could get to setting it globally. The only "global" timeout you can set is for the aggregate execution time of all the tests in the run. We set that too, but I figured that we'd still want a per-test timeout as well.

Doing this at the class level actually isn't even documented in the pytest-timeout documentation, but it works based on the documented behavior of general pytest mark decorators on classes. Note that the decorator on the class actually takes precedence over a decorator on an individual function, so we can't easily override individual test times this way...

Reply:
@pschorf - Thoughts?

Reply:
I'll move the magic `600` into a variable in `util`.